script.js

```diff
@@ -99,6 +99,67 @@ data.forEach(works => {
    </div>
     `;
     workSection.insertAdjacentHTML('beforeend', cardHTML);
-    // console.log(workSection);
+    console.log(workSection);
     counter += 1;
+});
+
+// Select cards button for popup
+
+const popupButtons = document.querySelectorAll('.btn-work');
+popupButtons.forEach(popBtn => {
+  popBtn.addEventListener('click', ()=> {
+    const btnWorkIndex = popBtn.dataset.works;
+    const {title, description, image, technologies} = data[btnWorkIndex];
+    
+    let listOfTechs = '';
+    technologies.forEach(tech => {
+    listOfTechs += `<li class="list">${tech}</li>`;
+  });
+
+  const modalpopup = `
+  <div class="modal-content">
+    <div id="close-moda" class="close-btn"> <p>X</p></div>
+    <div class="headline1">
+      <ul class="tonic">
+        <li><h3>${title}</h3></li>
+      </ul>
+      <ul class="card-head">
+        <li class="canopy1">CANOPY</li>
+        <li class="canopy">Back End Dev</li>
+        <li class="canopy">2015</li>
+      </ul>
+    </div>
+    <div class="card" id="card${counter+1}">
```

Comment:
- Good job so far with the popup model but it still needs some finishing touches 👍 
- [ ] Kindly style the mobile version of the pop-up window to look like the template. Right now it is taking only the center of the page.
- [ ] It would be great to disable the page scroll when the popup window is on the screen. This will increase the user interface of your page.
- [ ] Also kindly bring the scroll bar of the popup closer to the pop-up.
- Your site:
![Annotation 2022-03-12 194302](https://user-images.githubusercontent.com/85113567/158030949-e0c8f6b1-3db0-45d3-85d6-ce0405ecce7f.jpg)
- Figma template:
![Annotation 2022-03-12 194319](https://user-images.githubusercontent.com/85113567/158030957-34267e60-407a-4bfd-a443-0bee49f968ea.jpg)